### PR TITLE
[Docs] 'SSL_CTX_set_cert_store' ownership of 'store'

### PR DESCRIPTION
**Pull request by ruslo**
##### Checklist

- [x] documentation is added or updated

### CODE REVIEW

File: doc/man3/SSL_CTX_set_cert_store.pod [hunk 1]
```diff
@@ -16,7 +16,9 @@ SSL_CTX_set_cert_store, SSL_CTX_set1_cert_store, SSL_CTX_get_cert_store - manipu
 
 SSL_CTX_set_cert_store() sets/replaces the certificate verification storage
 of B<ctx> to/with B<store>. If another X509_STORE object is currently
-set in B<ctx>, it will be X509_STORE_free()ed.
+set in B<ctx>, it will be X509_STORE_free()ed. SSL_CTX_set_cert_store() will
+take ownership of the B<store>, i.e., the call X509_STORE_free(store) is no
```
Comment (t8m) [inline doc/man3/SSL_CTX_set_cert_store.pod:20]: `C<X509_STORE_free(store)>`

**Review by t8m — CHANGES_REQUESTED** [review]

File: doc/man3/SSL_CTX_set_cert_store.pod (hunk 1 above)
Comment (ruslo) [reply doc/man3/SSL_CTX_set_cert_store.pod]: Applied

**Comment by mattcaswell** [comment]
Please could you submit a CLA?

https://www.openssl.org/policies/cla.html

**Comment by ruslo** [comment]
> Please could you submit a CLA?

@mattcaswell I did it yesterday 👌 waiting for a reply

**Comment by ruslo** [comment]
> You have been added to our CLA database and should now be able to
contribute. If you have any open pull requests, you must close and reopen them to
remove the "needs CLA" tag.

**Review by t8m — APPROVED** [review]

**Review by mattcaswell — APPROVED** [review]

**Comment by t8m** [comment]
Merged to all the active branches. Thank you for your contribution.

File: doc/man3/SSL_CTX_set_cert_store.pod (hunk 1 above)
Comment (ruslo) [reply doc/man3/SSL_CTX_set_cert_store.pod]: > C<X509_STORE_free(store)>

@t8m Can you please confirm that it's rendered as expected? I see that now the `X509_STORE_free(store)` definitely stands out from other similar objects like `X509_STORE_free()` or `SSL_CTX_set_cert_store()`:

openssl_screen

- https://www.openssl.org/docs/manmaster/man3/SSL_CTX_set_cert_store.html

Comment (t8m) [reply doc/man3/SSL_CTX_set_cert_store.pod]: IMO this is correct. `X509_STORE_free(store)` is a code snippet where those functions are just names of functions.

**Pull request closed without merging**